### PR TITLE
Pulse autoplay changes

### PR DESCRIPTION
**Pull request by aeng7**
Updates the Pulse ad manager to allow for muted autoplay.

Note there are no unit tests for the Pulse ad plugin.

### CODE REVIEW

File: js/pulse.js [hunk 1]
```diff
@@ -1044,14 +1056,41 @@
...
 
+            /**
+             * Callback for when we receive the AD_VOLUME_CHANGED event from the Pulse SDK. We will ask
+             * the video controller wrapper to notify the player of the volume change event.
+             * @private
+             * @method Pulse#_onAdVolumeChanged
+             * @param {String} event The event name
+             * @param {Object} metadata The metadata associated with the event
+             */
+            var _onAdVolumeChanged = function(event,metadata) {
```
Comment (dallen42) [inline js/pulse.js]: Missing a space after the comma here

Comment (aeng7) [reply js/pulse.js]: Will add

**Comment by pilievOoyala** [comment]
LGTM

**Comment by aeng7** [comment]
@MehdiDadashPour @pberdys we're going to merge this today so that QA can get started testing with the limited days we have left before release. If you have any comments, we'll address them tomorrow. Please take a look at this PR when you can, thanks!

**Pull request merged**